### ISSUE TITLE
Consideration of data streams used for compliance

### ISSUE THREAD

**Issue by dtdougherty** [shcaba/FishPath-updates#282]
There are several caveats in FP that suggest caution around fisher willingness to cooperate, suggesting that false reporting may take place if fear of compliance violation. Somewhere in the tool or process we should note that if the same data is used for compliance and for assessments, alternative (backup) sources should be sought in case of mis-reporting. This should also be considered when developing any new enforcement protocols in the process, so that impacts on assessment are considered.

**Comment by NatalieDowling** [comment]
I agree. We could add your suggested text to the current caveats, or (maybe better) invoke new caveats.
